unit_tests/testVTUFiles/testVTUFiles.f90

```diff
@@ -49,92 +50,118 @@ SUBROUTINE SetupError
     ENDSUBROUTINE SetupError
 !
 !-------------------------------------------------------------------------------
-    SUBROUTINE testInitialize
-      !Test Initialize and clear
-      CALL testVTUFile%clear
+    SUBROUTINE testClear()
+      !Test clear
+      CALL testVTUFile%clear()
+      bool=.NOT.testVTUFile%isInit()
+      ASSERT(bool,'testVTUFile%clear(...)')
+    ENDSUBROUTINE testClear
+!
+!-------------------------------------------------------------------------------
+    SUBROUTINE testInitialize()
+      !Test initialize
       CALL testVTUFile%initialize(666,'testVTU0.vtu')
-      bool = testVTUFile%isInit() .AND. testVTUFile%isOpen()
+      bool=testVTUFile%isInit().AND.testVTUFile%isOpen()
       ASSERT(bool,'testVTUFile%initialize(...)')
```

Comment:
There should be more asserts here. e.g. after initialization does the object have a mesh? etc.

It's better to not combine assertions on attributes. This obfuscates the debugging process. I recommend something like:
```
ASSERT(testVTUFile%isInit(),'%isInit()')
ASSERT(testVTUFile%isOpen(),'%isOpen()')
```
etc.